### PR TITLE
Add names to analyser json

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: pkg/collectors/devices/pmc.go [hunk 1]
```diff
@@ -15,10 +15,11 @@ import (
 )
 
 type PMCInfo struct {
+	Timestamp               string `fetcherKey:"date"                    json:"timestamp" `
 	TimeSource              string `fetcherKey:"timeSource"              json:"timeSource"`
 	ClockAccuracy           string `fetcherKey:"clockAccuracy"           json:"clockAccuracy"`
 	OffsetScaledLogVariance string `fetcherKey:"offsetScaledLogVariance" json:"offsetScaledLogVariance"`
-	ClockClass              int    `fetcherKey:"clockClass"              json:"clockClass"`
+	ClockClass              int    `fetcherKey:"clockClass"              json:"clkcl"`
```
Comment (natifridman) [inline pkg/collectors/devices/pmc.go:22]: Not sure if `clkcl` is meaningful enough

**Review by natifridman — COMMENTED** [review]
Looks like this breaks the pmc test

File: pkg/collectors/devices/pmc.go (hunk 1 above)
Comment (nocturnalastro) [reply pkg/collectors/devices/pmc.go:22]: Good point I've added a comment to the PR in `vse-sync-pp` we need to match there names for the JSON object handling to work.

**Comment by nocturnalastro** [comment]
I fixed the tests and split the PMC changes into there own commit for clarity

File: pkg/collectors/devices/pmc.go (hunk 1 above)
Comment (natifridman) [reply pkg/collectors/devices/pmc.go:22]: https://github.com/redhat-partner-solutions/vse-sync-pp/pull/28#discussion_r1285621099

**Review by natifridman — APPROVED** [review]

**Pull request merged**